src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/DefineAccessorsForAttributeArgumentsTests.Fixer.cs

```diff
@@ -153,7 +153,6 @@ public InternalGetterTestAttribute(string name)
     public string Name
     {
         get { return m_name; }
-
         internal set { m_name = value; }
```

Comment:
Note: these are desirable changes.  the original code had the accessors abutting each other.  so adding an accessibility modifier should not change that.  this was a bug fix in roslyn to improve this behavior.